packages/@webex/internal-plugin-encryption/src/constants.js

```diff
@@ -0,0 +1,3 @@
+export const KMS_KEY_REVOKE_FAILURE = 'event:kms:key:revoke:encryption:failure';
+export const KMS_KEY_REVOKE_ERROR_STATUS = 405;
+export const KMS_KEY_REVOKE_ERROR_CODE = 405005 || 405006;
```

Comment:
this variable should be a list... 

right now this is saying 

if (405005 || 405006) { KMS_KEY_REVOKE_ERROR_CODE = 405005 } and thus it will always be equal to 405005. 

instead we'd want a list to check if VALID_ERROR_CODES.includes(errorCode)
or something of the like

Reply:
ah I see, got it. Let me change that